src/frontend/src/helpers/Services.ts

```diff
@@ -15,14 +15,22 @@ import {AbstractCompiler,
 
 export * from "./Storage/Interface";
 export * from "./Compiler/Interface";
-export {Services, LoginError, Connection, DispatchFunction};
+export {Services, GenericError, LoginError, Connection, DispatchFunction};
 
-class LoginError extends Error {
-  constructor(msg: string,
+class GenericError extends Error {
+  __proto__: Error;
+    constructor(message?: string) {
+        const trueProto = new.target.prototype;
+        super(message);
+    }
+}
+
```

Comment:
That's a fix to repair the prototype chain so instanceof works properly.  See #721.